integration_test/third_party_apps_data/applications/mongodb3.6/debian_ubuntu/install

```diff
@@ -9,6 +9,11 @@ osname="$(lsb_release -si)"
 
 case $osname in
   Ubuntu)
+    if [ "$oscode" = "lunar" ]; then
+      # MongoDB 3.6 requires libssl1, but it cannot be installed through apt on Ubuntu 23.04 (lunar)
```

Comment:
If MongoDB 3.6 is not supported beyond Ubuntu 18.04, should we even be testing it on later platforms?

Reply:
The current direction we've been given is to continue to support applications as widely as possible in the kokoro integration tests. If you want to discuss this plan further we should have time during the weekly status meeting today (Tuesday) that I believe you have an invitation to, would be happy to discuss further.